src/templates/en/2019/index.html

```diff
@@ -170,9 +130,9 @@ <h2 class="title title-alt">Contributors</h2>
     </div>
     <div class="people">
       <div class="people-number">{{ contributors }}</div>
-      <img id="character-markup" class="character" src="/static/images/character-markup.png" alt="" loading="lazy" />
-      <img id="character-star" class="character" src="/static/images/character-star.png" alt="" loading="lazy" />
-      <img id="character-hat" class="character" src="/static/images/character-hat.png" alt="" loading="lazy" />
+      <img id="character-markup" class="character" src="/static/images/character-markup.png" alt=""/>
```

Comment:
Was the lazy loading removed on purpose?

Reply:
Due to rebase & conflicts, will add again. Sorry.